script/lib/github.py

```diff
@@ -1,9 +1,11 @@
-#!/usr/bin/env python
+# Copyright (c) 2018 The Brave Authors. All rights reserved.
```

Comment:
not sure if this is the correct copyright here cc @bbondy

Reply:
LGTM - I put a commit through keeping the shebang. License header LGTM

Reply:
whoops- this caused a linting error. I'll back it out